build.gradle

```diff
@@ -25,6 +25,8 @@ import java.util.concurrent.TimeUnit
 allprojects {
     repositories {
         mavenCentral()
+        maven { url 'https://jitpack.io' }
+        maven { url 'https://maven.twttr.com' }
```

Comment:
We should move these additional repos to ParquetHadoop/build.gradle if they are truly necessary.

We should add comments about which repositories we are expecting to get our non-standard dependencies from.